x-pack/plugins/event_log/server/saved_object_provider_registry.ts

```diff
@@ -49,9 +56,16 @@ export class SavedObjectProviderRegistry {
     // `scopedProviders` is a cache of providers which are scoped t othe current request.
     // The client will only instantiate a provider on-demand and it will cache each
     // one to enable the request to reuse each provider.
-    const scopedProviders = new Map<string, SavedObjectGetter>();
+
+    // would be nice to have a simple version support in API:
+    // curl -X GET "localhost:9200/my-index-000001/_mget?pretty" -H 'Content-Type: application/json' -d' { "ids" : ["1", "2"] } '
+    const scopedProviders = new Map<string, SavedObjectBulkGetter>();
     const defaultGetter = this.defaultProvider(request);
-    return (type: string, id: string) => {
+    return (type: string, ids: string[]) => {
+      const objects = ids.reduce(
+        (prev: Array<{ type: string; id: string }>, id) => [...prev, { type, id }],
+        []
+      );
```

Comment:
nit: might be clearer to use a `map` instead of a `reduce`
`const objects = ids.map((id: string) => ({ type, id }));`